crates/rolldown_plugin_dynamic_import_vars/src/to_glob.rs

```diff
@@ -0,0 +1,118 @@
+// Ported from https://github.com/rollup/plugins/blob/944e7d3ec4375371a2e70a55ac07cab4c61dc8b6/packages/dynamic-import-vars/src/dynamic-import-to-glob.js
+
+use crate::should_ignore::should_ignore;
+use oxc::{
+  ast::ast::{Argument, BinaryExpression, CallExpression, Expression, TemplateLiteral},
+  syntax::operator::BinaryOperator,
+};
+use regex::Regex;
+use std::path::Path;
+use std::sync::LazyLock;
+
+// Disallow ./*.ext
+static OWN_DIRECTORY_STAR_REGEX: LazyLock<Regex> = LazyLock::new(|| {
+  let pattern = r"^\./\*\.\w+$";
+  Regex::new(pattern).expect("failed to compile regex")
+});
+
+static EXAMPLE_CODE: &str = "For example: import(`./foo/${bar}.js`).";
+
+pub fn to_glob_pattern(expr: &Expression) -> anyhow::Result<Option<String>> {
```

Comment:
Could you port some unit tests from https://github.com/rollup/plugins/blob/master/packages/dynamic-import-vars/test/src/dynamic-import-to-glob.test.mjs.